packages/v4/src/content/design-guidelines/components/form/forms.md

```diff
@@ -212,10 +218,9 @@ Some common use cases include:
 ### Validation on submission
 Validation on submission is performed when the user attempts to submit a form. In this scenario, the user completes the form in its entirety and the page is reloaded with any errors detected. 
 
-For validation on submission, use an [inline alert](/components/alert/design-guidelines#inline-alerts) along with field level errors wherever possible to make it simple for the user to quickly identify areas that need to be fixed.
-Notifications should describe the error and help users solve the problem.
+For validation on submission, use an [inline alert](/components/alert/design-guidelines#inline-alerts) along with field level errors. This combination of error types should make it simple for the user to quickly identify areas that need to be fixed. The inline alert should be placed below the form's header and description, and should take on the width of the form. Inline alerts should describe the general error, but refrain from going into great detail.
 
-<img src="./img/error-validation.png" alt="Example of errors after validation on submission, including an inline alert at the top, and field level errors" width="459"/>
+<img src="./img/inline_error.png" alt="Example of errors after validation on submission, including an inline alert at the top, and field level errors" width="551"/>
```

Comment:
Add little red star next to "state of residence" to show that it's a required field